.github/workflows/test-k3d.yml

```diff
@@ -22,7 +22,12 @@ jobs:
             ~/go/pkg/mod
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
-            ${{ runner.os }}-go-        
+            ${{ runner.os }}-go-
+
+      - name: "Install K3d and deploy a cluster"
+        run: |
+          wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
```

Comment:
Thoughts on using a specific k3d release tag (like we did w/ KinD)? 

It might be annoying to debug why our tests are failing if some breaking update happens in k3d main.

Reply:
That's a good point I was just copy/pasting both of their official docs

Reply:
So I did make them consistent, but went the opposite direction.  I agree this could pose a more flaky CI/Cd, but it now matched how https://zarf.dev/install/ works and how a real user would experience Zarf, so that matches better with the change to Journey tests I think.  I'd rather have the test break here and slow our debugging down than break for a user.